### PR TITLE
Added Metric: Number of Open TCP Connections.

### PR DESCRIPTION
**Pull request by tahteche**
Added metric to display the number of open TCP connections in the mod_admin_statistics module.

### CODE REVIEW

File: modules/mod_admin_statistics/templates/admin_statistics.tpl [hunk 1]
```diff
@@ -70,7 +75,10 @@ function to_human(value) {
 {% endjavascript %}
 
 {% javascript %}
-    pubzub.subscribe("erlang/stats/#", function(topic, msg) {
+    pubzub.subscribe("erlang/stats/#", function(topic, msg) {   
+        //print topics to log to see all topics available under erlang/stats/#
+        console.log(topic, msg);
```
Comment (mworrell) [inline modules/mod_admin_statistics/templates/admin_statistics.tpl]: Can you remove this console.log?
On certain browsers (IE) it is not available and will crash the Javascript.

Comment (tahteche) [inline modules/mod_admin_statistics/templates/admin_statistics.tpl]: Thanks for the heads-up. Its removed.

Comment (mworrell) [inline modules/mod_admin_statistics/templates/admin_statistics.tpl]: :+1:

**Comment by arjan** [comment]
This looks great to me. @mmzeeman could you also check this out? This is part of @tahteche 's proposal for his GSoC 2016 project.

**Comment by mmzeeman** [comment]
Yes, this is nice work. I have also read the proposal which is also good. I can merge the pull after I'm done with an upgrade which is scheduled around 23:00

Verstuurd vanaf mijn iPhone

**Comment by mmzeeman** [comment]
Thanks :+1:

**Pull request merged**